server/routes/profileRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const profileController = require("../controllers/profileController");
+
+router.post("/:id", profileController.createProfile);
```

Comment:
We don't need the `:id` parameter here. You can instead obtain the logged in user's id from the request object.

Eg: `req.user.id` - make sure you are using the `protect` middleware on your routes